fuel-vm/src/checked_transaction.rs

```diff
@@ -1606,8 +1703,14 @@ mod tests {
     where
         Tx: Chargeable + field::Inputs + field::Outputs,
     {
-        let available_balances =
-            balances::initial_free_balances(tx, gas_costs, fee_params, base_asset_id)?;
+        let arb_gas_price = 1;
```

Comment:
Why did we stop using quickcheck to set the gas price?

Reply:
Oh. I thought I fixed these spots. @xgreenx pointed it out but looks like I missed this one. Thanks.